src/lib/core/CHIPError.h

```diff
@@ -28,13 +28,13 @@
 
 #pragma once
 
-#include <lib/core/CHIPConfig.h>
-#include <lib/support/TypeTraits.h>
-
 #include <inttypes.h>
 #include <limits>
 #include <type_traits>
 
+#include <lib/core/CHIPConfig.h>
```

Comment:
Blocking comment to check why `this PR - c++20` has bigger .data section than `master - c++20`...